recipe/meta.yaml

```diff
@@ -1,33 +1,32 @@
-{% set version = "4.0.1" %}
+{% set version = "6.0.0" %}
 
 package:
   name: flake8
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/flake8/flake8-{{ version }}.tar.gz
-  sha256: 806e034dda44114815e23c16ef92f95c91e4c71100ff52813adf7132a6ad870d
+  sha256: c61007e76655af75e6785a931f452915b371dc48f56efd765247c8fe68f2b181
 
 build:
-  number: 1
-  noarch: python
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - flake8 = flake8.main.cli:main
 
 requirements:
   host:
-    - python
+    - python >=3.8.1
     - pip
     - wheel
     - setuptools >=30.0.0
   run:
-    - python >=3.6
+    - python >=3.8.1
     # mccabe 0.7.0 is a noarch python package now and it hasn't broken changes for flake8
-    - mccabe >=0.6.0,<0.8.0
-    - pycodestyle >=2.8.0,<2.9.0
-    - pyflakes >=2.4.0,<2.5.0
-    - importlib-metadata
+    - mccabe >=0.7.0,<0.8.0
+    - pycodestyle >=2.10.0,<2.11.0
+    - pyflakes >=3.0.0,<3.1.0
+
 test:
```

Comment:
Is `pip check` still failing?

Reply:
After a few tweaks to the skip section and run section, `pip check` now functions properly!

Reply:
Then the comment above it doesn't make sense anymore